engine-api/src/test/java/io/nosqlbench/engine/api/util/TagFilterTest.java

```diff
@@ -110,7 +106,8 @@ public void testMatchingKeyAndValueRegexDoesMatch() {
         assertThat(tfRight.matches(itemtags).matched()).isTrue();
     }
 
-    public void testMatchingDetails() {
+    // TODO: enable and fix test
```

Comment:
i wasnt sure how this is supposed to work, enabling the test caused it to fail

Reply:
I've updated the test. The flow has changed since the test was originally written with an earlier test framework that didn't require per-method annotations.
The change in flow was to keep diag logs more concise without notifying the user of additional predicate misses once one has already missed.